documentation/src/docs/asciidoc/release-notes/release-notes-5.6.0-M1.adoc

```diff
@@ -29,6 +29,8 @@ on GitHub.
 * New `printFailuresTo(PrintWriter, int)` method in `TestExecutionSummary` that allows one
   to specify the maximum number of lines to print for exception stack traces.
 
+* New `TypeBasedParameterResolver<T>` - a generic adapter of `ParameterResolver` - that eases the creation
+  of custom resolver by providing a type-based `supportsParameter` default implementation.
```

Comment:
:x: Please remove the extra blank line above the item and wrap lines at 90 characters.